pkgs/jnigen/example/in_app_java/lib/android_utils.dart

```diff
@@ -369,10 +369,14 @@ class EmojiCompat_Config extends jni.JObject {
   }
 
   static final _getMetadataRepoLoader = jniLookup<
-              ffi
-              .NativeFunction<jni.JniResult Function(ffi.Pointer<ffi.Void>)>>(
-          "EmojiCompat_Config__getMetadataRepoLoader")
-      .asFunction<jni.JniResult Function(ffi.Pointer<ffi.Void>)>();
+          ffi.NativeFunction<
```

Comment:
Happy to rubberstamp this formatting change in a separate PR to make this PR smaller.